### PR TITLE
docs: rewrite top page for first release

### PR DESCRIPTION
**Pull request by MH4GF**
Only the top page was seriously rewritten.

- **🔧 Add lucide-react dependency to package.json**
- **🔧 Add 'Roadmap' link to layout configuration**
- **docs: re-write the docs index page**

### CODE REVIEW

File: frontend/apps/docs/content/docs/index.mdx [hunk 1]
```diff
@@ -1,44 +1,68 @@
...
 
 ## How to Get Started
 
-Choose one of the following workflows depending on your needs:
+### Public Project Setup
+
+Want a quick setup for a public repository? Just drop the URL of your schema file!
+
+<input />
+<button>Generate</button>
```
Comment (MH4GF) [inline frontend/apps/docs/content/docs/index.mdx:49]: 📝 Implemented in another issue

**Review by sasamuku — APPROVED** [review]
LGTM🚀

File: frontend/apps/docs/app/layout.config.tsx [hunk 2]
```diff
@@ -14,4 +15,11 @@ export const baseOptions: BaseLayoutProps = {
     url: '/docs',
   },
   githubUrl: 'https://github.com/liam-hq/liam',
+  links: [
+    {
+      icon: <FolderKanban />,
+      text: 'Roadmap',
+      url: 'https://github.com/orgs/liam-hq/projects/1',
```
Comment (sasamuku) [inline frontend/apps/docs/app/layout.config.tsx:22]: 👍✨

File: frontend/apps/docs/content/docs/index.mdx [hunk 3]
```diff
@@ -1,44 +1,68 @@
...
+
+Curious about what Liam ERD can do? Visit https://liambx.com/erd/galaxy to browse a variety of example ER diagrams from real-world schemas.
+
+## Supported Schemas
+
+Legend:
+
+- ✅: Supported
+- ❌: Not in progress
+- ⌛️: In progress
```
Comment (hoshinotsuyoshi) [inline frontend/apps/docs/content/docs/index.mdx:30]: ❤️

File: frontend/apps/docs/content/docs/index.mdx [hunk 4]
```diff
@@ -1,44 +1,68 @@
...
+## Explore Diagrams
+
+Curious about what Liam ERD can do? Visit https://liambx.com/erd/galaxy to browse a variety of example ER diagrams from real-world schemas.
+
+## Supported Schemas
+
+Legend:
+
+- ✅: Supported
+- ❌: Not in progress
```
Comment (sasamuku) [inline frontend/apps/docs/content/docs/index.mdx:29]: [nits]
❌ looks like we are not going to support this.
⛔ is more suitable? I'm not sure native sense of emoji 😅

File: frontend/apps/docs/content/docs/index.mdx [hunk 5]
```diff
@@ -1,44 +1,68 @@
...
+<button>Generate</button>
+
+For detailed instructions, check out [URL-based access](/docs/url-based-access).
+
+### Private Project Setup
+
+For internal or private repositories, run this command to start an interactive setup:
+
+```package-install
+npx @liam-hq/cli init
```
Comment (hoshinotsuyoshi) [inline frontend/apps/docs/content/docs/index.mdx:58]: Not implemented now, but it seems to be ok.

**Review by hoshinotsuyoshi — APPROVED** [review]
LGTM!

File: frontend/apps/docs/content/docs/index.mdx (hunk 4 above)
Comment (MH4GF) [reply frontend/apps/docs/content/docs/index.mdx:29]: Looks great!! ⛔

Comment (MH4GF) [reply frontend/apps/docs/content/docs/index.mdx]: fixed: https://github.com/liam-hq/liam/pull/366/commits/5bc68d5c16bbff9faf969ac549a8c8e1f15f4548

**Pull request merged**